.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
Should this file be included, if the e2e tests are run in the Travis workflow? Just wondering if we're intending to run the e2e tests in Travis and in GitHub actions

Reply:
Since the plan is to move to GitHub actions, my plan is to just keep this as a placeholder for now.  It won't execute since I made sure to not configure the branch names until we are ready to go.  Once it's working I'll remove Travis.